arch/arm/arm/include/uk/asm/compiler.h

```diff
@@ -0,0 +1,35 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/*
+ * Copyright (c) YYYY, Copyright Holder. All rights reserved.
```

Comment:
Please, fix copyright information.

Reply:
Actually this is intentional, because these headers are empty, so I think it doesn't make sense to grant the copyright to myself. The idea was that the copyright should be updated by whoever adds content. This is also what I do in the next commit where I add come content to `arch/arm/arm64/include/uk/asm/compile.h` and grant the copyright to myself.